cjwkernel/thrift.thrift

```diff
@@ -192,6 +192,22 @@ union I18nArgument {
   3: double double_value
 }
 
+/** Source of a translatable string. 
+    If none of the values are set, this means it's coming from workbench itself
+*/
+union I18nMessageSource {
+  /**
+   * If `"library"` is set, this means it's coming from some of our supported libraries
```

Comment:
Nix the `"If ... is set, this means it's coming from"` in these two comments. (The reader must read them all to determine they can all be ignored.) Instead:

```
/** The message comes from the "library" library */
...
/** The message comes from the "module_id" module */
```